flume/doc/quick_start_guide.md

```diff
@@ -36,12 +36,6 @@ EOL
 $ sudo yum install cygnus
 ```
 
-When installing through the RPM, Cygnus is started as a service. Stop it since this Quick Start Guide will run Cygnus from the command line.
```

Comment:
Why this have been removed? Maybe Cygnus is not started as service as a consecuence of `yum install cygnus`? (it would be fine... only curiosity).